lms/templates/student_account/login_and_register.html

```diff
@@ -27,3 +27,4 @@
 <div class="section-bkg-wrapper">
     <div id="login-and-registration-container" class="login-register" />
 </div>
+<%include file="../register_cme/form.html" />
```

Comment:
Actually no, take a look at the file. It only does something if the flag is enabled.